code/src/Components/FirstQ.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+const FirstQuestion = ({name, setName,  nextSection, handleSubmit}) => {
+    return (
+        
+            <article >
+                <div className="text-group">
+                    <h1>Welcome to the music survey</h1>
+                    <label htmlFor="name" 
+                    key="name" 
+                    onSubmit={handleSubmit}>
+                    Enter your name
+                        <input 
+                        className="name-input"
+                        type="text" 
+                        name="name" 
+                        onChange={(event)=>setName(event.target.value)} 
+                        value={name} />
+                    </label> 
+                </div>
+
+                <div className="btn-group">
+                <button className="next-btn" disabled={name === ""}
```

Comment:
Wow, didn't know about this disabled-thing. Great that the user can't push the button without typing his/her name in the field, and also that your button turns dark pink when writing the name in the field